### PR TITLE
Style Work page.

### PR DESCRIPTION
**Pull request by mathewjordan**
## What does this do?

This work brings in some styling decisions on the Work page that should influence the direction of Digital Collections. Along with it, colors and fonts are beginning to be incorporated more intently. Also added is the Northwestern SVG logo.

image

In general, decisions made are:

- Take advantage of whitespace/negative areas and let stuff breath
- Big bold titles using **Campton Bold** for Works, Collections, and Related links
- Utilize purple (below the header) more intently as an **action** color for links and buttons
- Tick up the font size for body content

### CODE REVIEW

File: components/Shared/DefinitionList.styled.ts [hunk 1]
```diff
@@ -3,15 +3,17 @@ import { styled } from "@/stitches.config";
...
 
 const DefinitionListWrapper = styled("div", {
+  lineHeight: "1.47em",
+
   "& dt": {
-    fontSize: "0.75rem",
-    fontWeight: "bold",
-    paddingBottom: "0.5rem",
-    textTransform: "uppercase",
+    fontSize: "$3",
```
Comment (adamjarling) [inline components/Shared/DefinitionList.styled.ts:9]: Great, I've been looking for these.

File: components/Work/TopInfo.styled.ts [hunk 2]
```diff
@@ -5,26 +5,77 @@ import { styled } from "@/stitches.config";
...
+      color: "$black50",
+      fontFamily: "$sansLight",
+      lineHeight: "1.47em",
+    },
+  },
+
+  [`> div`]: {
+    display: "grid",
+    gap: "$7",
+    gridTemplateColumns: "618fr 382fr",
```
Comment (adamjarling) [inline components/Work/TopInfo.styled.ts:62]: Wow, never seen `fr` values this big!   Can you explain a little why?

File: components/Work/ViewerWrapper.tsx [hunk 3]
```diff
@@ -2,21 +2,56 @@ import React from "react";
...
+}> = dynamic(() => import("@samvera/clover-iiif"), {
+  ssr: false,
+});
 
 interface WrapperProps {
   manifestId: string;
 }
 
 const WorkViewerWrapper: React.FC<WrapperProps> = ({ manifestId }) => {
+  const customTheme = {
```
Comment (adamjarling) [inline components/Work/ViewerWrapper.tsx:21]: Nice

File: styles/global.ts [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+    },
+  ],
+};
+
+const globalStyles = globalCss({
+  ...defaults,
+  ...fonts,
+});
+
+export default globalStyles;
```
Comment (adamjarling) [inline styles/global.ts:84]: Wow, this is cool.  So this is the Stitches way of injecting globals eh?   Seems clean to me.

**Review by adamjarling — APPROVED** [review]
This is fantastic work.  Looks great!

File: styles/global.ts (hunk 4 above)
Comment (mathewjordan) [reply styles/global.ts:84]: Yup, just discovered this. Its not well documented and took some trial and error. Seems like a good idea for a medium article.

File: components/Work/TopInfo.styled.ts (hunk 2 above)
Comment (mathewjordan) [reply components/Work/TopInfo.styled.ts:62]: It's a golden ratio calculation, not absolutely necessary.

**Pull request merged**